### DISCUSSION TITLE
Feature requests for ACE Anvil

### DISCUSSION THREAD

**Discussion by BaerMitUmlaut** [acemod/ACE-Anvil#1] [Announcements]
### tl;dr: Post your feature requests here in the pinned post of the [feature requests category](https://github.com/acemod/ACE-Anvil/discussions/categories/feature-requests).

---

## The problem of feature requests
At the point of writing, we have received around 2000 feature requests for ACE3. Of these 2000 requests, many are too complex or simply not possible in Arma, some might be duplicates, but within this giant mass of ideas there will also be some gems which we might miss. Because of this, we want to give you a better opportunity to give your ideas some visibility.

## Introducing feature request voting
For ACE Anvil, we want to try something new. GitHub's discussions allow you to upvote comments, which allows the community to vote on everyone's ideas. If you have an idea that you want to share, post a new comment on the current feature request thread - and while you're at it, upvote other ideas you like. To keep things manageable, we will post a new thread with every major release. This way, new ideas don't get drowned out and have a fair chance to land at the top.

## What this doesn't mean
If your idea becomes the most upvoted comment on a feature request thread, it doesn't mean that the team will immediately start working on that idea. In fact, it might never be implemented. ACE still is a hobby project to us and none of us get paid to work on it. The main purpose of voting is to allow the team to get a better idea of what features are most desired.
